### ISSUE TITLE
Android Basics: Activities and intents

### ISSUE THREAD

**Issue by joharris18** [google-developer-training/android-basics-kotlin-words-app#179]
**URL of codelab**
https://github.com/google-developer-training/android-basics-kotlin-words-app/tree/starter

**In which task and step of the codelab can this issue be found?**
starter code

**Describe the problem**
I am not sure if I am just confused but but the starter code is showing two kotlin code fragments (words and letter)
instead of the 'DetailActivity.kt' which is mention on the codelab. It makes it hard to follow through since it is different.

**Steps to reproduce?**
1. Go to...
2. Click on...
3. See error...

**Versions**
_Android Studio version:_ 
_API version of the emulator:_ 

**Additional information**
_Include screenshots if they would be useful in clarifying the problem._

![image](https://user-images.githubusercontent.com/91049108/133965860-6d426426-19c5-47ff-ba1d-082c9f971345.png)

**Comment by calren** [comment]
This seems like an issue with Github that has now been fixed (the issue was the attempting to download a zip of a branch was instead downloading a zip of the `main` branch, which is the completed version of the codelab). I've tried download the zip file of the starter branch and confirmed it was not the completed app.
